hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdb/util/TestManagedSstFileReader.java

```diff
@@ -58,7 +62,7 @@ class TestManagedSstFileReader {
 
   // Key prefix containing all characters, to check if all characters can be
   // written & read from rocksdb through SSTDumptool
-  private static final String KEY_PREFIX = IntStream.range(0, 256).boxed()
+  private static final String KEY_PREFIX = IntStream.range(1, 256).boxed()
```

Comment:
What is the justification for the change from `0` to `1`?

Reply:
As part of the bounds we pass utf-8 strings. So string containing '\0' does not have the same value when encode UTF-8 on java which is 0. But in jni the utf-8 encoded value for '\0' becomes -64 -128. Thus the value becomes different. In order to support this, I have made the change on the rocks-tools to pass the address of the ManagedSlice we can use the object of the slice directly from there.

Reply:
It is a hacky solution but it works. Please review and let me know if I could do something better.

Reply:
Got it. Pls put this comment in the test class as well.

Reply:
+1 on adding comment for this.

Reply:
done